content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,175 @@
+<br/><br/>
+2. Navigate to the chat screen. There you should see the pending wallet notification. Click the "View config" button.<br/><br/>
+
+<img src = laptopConfigNote.png
+    alt = "pending keys config note (Screenshot)">
+
+3. Here you see the number of current keys assigned among other information about the wallet. Click the "Continue to assign keys" button.<br/><br/>
+
+<img src = laptopWalletConfig.png
+    alt = "wallet configuration (Screenshot)">
+
+4. You can now assign the key stored locally on your laptop. Notice that the other two keys are considered foreign and your local key is recognized as a "SOFTWARE" key.<br/><br/>
+
+<img src = laptopAssignKey.png
+    alt = "assign key configuration (Screenshot)">
+
+5. After assigning your key, click the "Continue" button. You should now be returned to the chat screen and see that your config notification is now only pending 1 key.<br/><br/>
+
+<img src = laptopPendingKeys.png
+    alt = "pending keys (Screenshot)">
+
+6. Continue to your next device!
+
+{{< /tab >}}
+
+{{< tab "Mobile" >}}
+
+{{< notice note >}}
+If you did not have all your keys available for the initial setup, you can continue from another device to continue the setup process. It is assumed that you have started to create a wallet. If you still have a pending configuration notification in your chat screen you can continue on your laptop here. If you do not, you should refer to the desktop tab to start a configuration.
+{{< /notice >}}
+
+1. Open your Nunchuk mobile app and navigate to the chat screen. You should see your wallet config notification here waiting for the last key!<br/><br/>
+
+<img src = mobilePendingKey.png
+    alt = "mobile app chat screen (Screenshot)">
+
+2. Click on the "View config" button.<br/><br/>
+
+3. Click the "Continue to assign keys" button at the bottom of the screen.<br/><br/>
+
+<img src = mobileContinueAssignKeys.png
+    alt = "Wallet configuration (Screenshot)">
+
+4. You can now assign the last key (in my case, it is the Mobile Key).<br/><br/>
+
+5. Click the "Assign keys and send invite" to finalize your wallet with the last key.<br/><br/>
+
+<img src = mobileAssignKeysSendInvite.png
+    alt = "assign keys and send invite (Screenshot)">
+
+6. You should be returned to the chat screen. You should now see the wallet config notification says "Pending finalization". On any of your devices you can open the config again by clicking the "View Config" and then the "Finalize wallet" button!<br/><br/>
+
+<img src = mobileAllKeysAssigned.png
+    alt = "mobile phone chat screen (Screenshot)">
+
+7. You will now be able to see the finalize button. When you click that button you will see all of the keys associated with the wallet and other important information. Click the "Finalize wallet" button once you're sure everything is correct.<br/><br/>
```

Comment:
Shorten this to:

Review the wallet configuration one more time, make sure everything is correct, then click the "Finalize wallet" button.